llvm/utils/lit/lit/TestRunner.py

```diff
@@ -1352,7 +1352,7 @@ def getDefaultSubstitutions(test, tmpDir, tmpBase, normalize_slashes=False):
 
     substitutions.append(("%{pathsep}", os.pathsep))
     substitutions.append(("%basename_t", baseName))
-    
+
```

Comment:
This looks suspicious. Did zou change the whitespace somehow?

Reply:
Oh, my editor is configured to strip trailing whitespace on save. Would you like me to undo this line?

Reply:
Ah, that didn't show up on my phone. It just showed that you changes the line. That's fine then I think.